sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/models/EncryptionAlgorithm.java

```diff
@@ -42,21 +67,41 @@
      * Static value A192CBC_HS384 for EncryptionAlgorithm.
      */
     public static final EncryptionAlgorithm A192CBC_HS384 = fromString("A192CBC-HS384");
-    
+
     /**
-     * Static value A256CBC for EncryptionAlgorithm.
+     * Static value A256CBC_HS512 for EncryptionAlgorithm.
      */
-    public static final EncryptionAlgorithm A256CBC = fromString("A256CBC");
-    
+    public static final EncryptionAlgorithm A256CBC_HS512 = fromString("A256CBC-HS512");
```

Comment:
These aren't in the list to implement. Just AES-CBC, AES-CBCPAD, AES-KW, and AES-GCM.

Reply:
Those were already made public in a previous release to provide feature-parity with the Track 1 SDK, I don't think we should remove them.

Reply:
Understood (.NET track 1 did too), but they aren't in the swagger. My understanding was that they were proposed and may indeed be implemented, but never made it into the public swaggers. @AlexGhiondea, @schaabs what do you think here? If we're going with parity with track 1, adding support for AES-HMAC using SHA2 makes sense, but are we doing that now?